xml/ha_migration.xml

```diff
@@ -107,7 +107,7 @@
     <listitem>
```

Comment:
Just found a minor bump in the above list with the following term:

```
Update
Installation of a newer minor version of a package, which usually contains security fixes
and important fixes.
```

the **security fixes and important fixes** sounds a bit strange to me. That would imply that security fixes are not important (is this is an exclusive and?). :wink:  What about "security and other fixes"?

Reply:
Good point. :) Changed to 'security fixes and other important fixes'.